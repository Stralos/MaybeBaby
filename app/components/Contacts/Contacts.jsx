/* @flow */
import React, { Component } from 'react';
import { Link } from 'react-router-dom';

export default class Contacts extends Component<{}> {
  onClick = () => {
    console.log('My Phone number is 862322479');
  }

  render() {
    return (
      <div>
        You can contact me by pressing this button!
        <Link to="/"> Home </Link>
        <button onClick={this.onClick}> Get my phone number</button>
      </div>
    );
  }
}
