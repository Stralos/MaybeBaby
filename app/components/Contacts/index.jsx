/* @flow */
import React, { Component } from "react";

export default class Contacts extends Component {
  onClick = () => {
    alert("My Phone number is 862322479")
  }

  render(){
    return (
      <div>
        You can contact me by pressing this button!
        <button onClick={this.onClick}> Get my phone number</button>
      </div>
    )
  }
}