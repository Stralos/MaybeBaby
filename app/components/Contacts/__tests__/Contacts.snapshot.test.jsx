/* @flow */
import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';

import Contacts from '../Contacts';

it('renders correctly', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <Contacts />
    </MemoryRouter>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
