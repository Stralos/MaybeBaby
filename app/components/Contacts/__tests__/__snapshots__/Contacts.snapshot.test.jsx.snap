// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  You can contact me by pressing this button!
  <a
    href="/"
    onClick={[Function]}
  >
     Home 
  </a>
  <button
    onClick={[Function]}
  >
     Get my phone number
  </button>
</div>
`;
